app/src/main/java/org/systers/mentorship/view/activities/RequestDetailActivity.kt

```diff
@@ -165,7 +165,6 @@ class RequestDetailActivity : BaseActivity() {
     }
 
     private fun setObservables(relationResponse: Relationship) {
```

Comment:
```suggestion
    private fun setObservables() {
```

No need to have the `Relationship` as a parameter here. The argument you are using on line 47 is already a global variable.

Reply:
Change line 47 form
`setOnClickListeners(mentorshipRelationResponse)`
to 
`setOnClickListeners()`

Reply:
Change line 174 from 
`previousScreen.putExtra(Constants.REQUEST_ID, relationResponse.id.toString())`
to 
`previousScreen.putExtra(Constants.REQUEST_ID, mentorshipRelationResponse.id.toString())`